syntax = "proto3";

package messages;

service LoggingService {
  rpc LogMessage (LogRequest) returns (LogResponse);
  rpc GetMessages (Empty) returns (MessageList);
}

message LogRequest {
  string id = 1;
  string txt = 2;
}

message LogResponse {
  string status = 1;
}

message Empty {}

message MessageList {
  repeated string messages = 1;
}